package.json

```diff
@@ -202,18 +203,17 @@
     "source-map-explorer": "2.5.3",
     "storybook-addon-pseudo-states": "2.1.0",
     "ts-jest": "29.1.2",
-    "ts-node": "10.9.2",
     "tsx": "4.7.2",
     "typescript": "5.4.5",
     "typescript-eslint": "7.3.0"
   },
   "volta": {
     "//node-comment": "Keep this in sync with 'engines' config at the top of the current file",
     "node": "20.11.0",
-    "//yarn-comment": "Keep this in sync with GitHub actions pipelines",
-    "yarn": "4.0.2"
+    "//bun-comment": "Keep this in sync with GitHub actions pipelines",
+    "bun": "1.1.8"
```

Comment:
issue: doesn't look like Volta supports Bun yet https://github.com/volta-cli/volta/issues/1465

Reply:
Can you try volta install bun? It worked on mac

Reply:
It worked on Windows too (had to use Admin privileges when running the command). The app started without issues as well. Should we wait for official support then? If not, things might break once it's added (and it works in a different way). 🤔

Reply:
I guess we will fix it, if it breaks when the official lease lands :-)